src/containers/Subject.js

```diff
@@ -266,29 +266,92 @@ class Subject extends Component {
 
     let launchPadRowData;
     if (Array.isArray(gplReport) && gplReport.length) {
+      // found existing GPL report, just show it
       launchPadRowData = {
         subject_id: subject.id,
         ...gplReport[0],
       };
     } else {
-      try {
-        const init = {
-          headers: { 'Content-Type': 'application/json' },
-          body: {
-            subject_id: subject.id,
-          },
-        };
-        const data = await API.post('gpl', '', init);
-        launchPadRowData = {
-          subject_id: subject.id,
-          ...data,
-        };
-      } catch (e) {
+      // NOTE: a bit of quick & dirty crude logic implementation here
+      // don't expect this to be stay longer after GPL ported into ICA
+      // as this should better be part of ICA Pipeline Workflow Automation like everyone else
+      // The need of on-demand "LaunchPad" feature is different thing -- i.e. to be considered
+      // better in Portal v2 revamp from ground up! Or, elsewhere dashboard. We shall see...
```

Comment:
FYI @scwatts @reisingerf @williamputraintan -- I will push this through for now. Take it as just another prototype idea / MVP to discuss further later on. :)